pkgs/development/python-modules/bash-kernel/default.nix

```diff
@@ -45,8 +45,16 @@ buildPythonPackage rec {
     ${python.pythonOnBuildForHost.interpreter} -m bash_kernel.install --prefix $out
   '';
 
-  # no tests
-  doCheck = false;
+  checkPhase = ''
+    # Create a JUPYTER_PATH with the kernelspec
```

Comment:
maybe add the `runHook preCheck` / `runHook postCheck`. Dunno how it could matter but it helps keep some consistency.